mne/viz/evoked.py

```diff
@@ -158,9 +185,9 @@ def _plot_evoked(evoked, picks, exclude, unit, show,
     channel_types = ['eeg', 'grad', 'mag', 'seeg']
 
     if picks is None:
-        picks = list(range(evoked.info['nchan']))
+        picks = list(range(info['nchan']))
 
-    bad_ch_idx = [evoked.ch_names.index(ch) for ch in evoked.info['bads']
+    bad_ch_idx = [evoked.ch_names.index(ch) for ch in info['bads']
```

Comment:
if the goal is to remove `evoked` from the calls, you can use `info['ch_names'].index(...)` instead